src/commands/docs/migrate.ts

```diff
@@ -94,7 +94,7 @@ export default class DocsMigrateCommand extends BaseCommand<typeof DocsMigrateCo
     }
 
     validationHookResults.successes.forEach(success => {
-      if (success.result && success.result.pages.length) {
+      if (success.result?.pages.length) {
```

Comment:
It recommended, but thankfully didn't autofix, that we move this to optional chaining.